scripts/setup-branches.sh

```diff
@@ -11,5 +11,5 @@ while IFS= read -r line
 do
```

Comment:
We don't need this file anymore, right? We just use the same logic in the update script afaict. Would you mind removing this if that's the case?

Reply:
Yep, I meant to do that! Probably forgot. I'll add the change shortly.